nvm.sh

```diff
@@ -1828,6 +1828,17 @@ nvm() {
         NVM_PREFIX="$(nvm_node_prefix)"
         NVM_SUCCESS_MSG="Uninstalled node $VERSION"
       fi
+
+      # Check version dir permissions
+      local INVALID_FILES="$(find "$VERSION_PATH" -exec [ ! -w "{}" ] \; -exec echo "{}" \;)"
+      if [ "$INVALID_FILES" ]; then
+        >&2 echo 'Cannot uninstall, incorrect permissions on installation folder.'
+        >&2 echo 'This is usually caused by running `npm install -g` as root. Run the following command as root to fix the permissions and then try again.'
+        >&2 echo
+        >&2 echo "  chown -R $(whoami) $VERSION_PATH"
```

Comment:
nit: the VERSION_PATH should probably be in quotes, and the command surrounded by backticks, in case it has spaces.